src/pages/ContactUs.js

```diff
@@ -1,9 +1,172 @@
+                        [theme.breakpoints.down('sm')]: {
+                            height: "250px",
+                            width: "435px",
+                            // width: "625px",
+                        },
+                    },
+                },
+            },
+        },
+
+    },
+}));
+
+function DonationComponent() {
+    const { ref, inView } = useInView({
+        triggerOnce: true,
+        rootMargin: "-150px",
+    });
+    return (
+        <Grid container sx={{mt:{ lg:8, md:4, sm:4, xs:4} }} ref={ref}>
+            {/*make this box invisible past a certain size*/}
+            <Grid item lg={5} sx={{position:'relative', display: {lg: 'flex', md: 'none', sm:'none', xs:'none' }}} justifyContent='flex-end' container>
+                <Box sx={{position: 'absolute', bottom:0, left:0, width: '100%', height: '52%', backgroundColor: "#f5f5f5"}}></Box>
+                <ImageTest sx={{bottom:{lg:25, md:25, sm:0}, position:'relative'}} style={{backgroundImage: `url(${group_3})`}} className={inView ? "slideIn1300ms" : "hidden"} />
+            </Grid>
+            <Grid item lg={7} md={12} sm={12} xs={12} className={inView ? "slideIn1300ms" : "hidden"}>
+                <Grid item container sx={{height:{lg:'48%'}, mt:{lg:0, md:3} }} justifyContent='center'>
+                    {/*the justfiy center is fine to leave as it does not come into use until we reach screen sm*/}
+                    <Grid item md={4} sm={8} xs={11} sx={{display: {lg: 'none', md: 'flex', sm:'flex', xs:'flex' }}} justifyContent='center'>
+                        <ImageTest style={{backgroundImage: `url(${group_3})`, borderRadius:'10px'}} className={inView ? "slideIn1300ms" : "hidden"} />
+                    </Grid>
+                    <Grid item lg={11} md={6} sm={11} xs={12} sx={{p:{lg:0, md:3, sm:3, xs:3} , pb:{lg:0, md:3, sm:5}, ml:{lg:0, md:1}}}>
+                        <Box sx={{height:'100%', flexDirection: 'column', display: 'flex', justifyContent:'center',alignItems:'flex-start'}}>
+                            <SectionTitle sx={{textAlign:'left'}}>How to support</SectionTitle>
+                            <SectionDescription sx={{mt:2}}>
+                                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                                Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+                                Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
+                            </SectionDescription>
+                        </Box>
+
+                    </Grid>
+                </Grid>
+
+                <Grid item container justifyContent='center' sx={{p:{lg:3, md:3, sm:3, xs:3}, pl:{lg:0, md:10}, mt:{lg:0, md:5}, pb:{lg:3, md:5, sm:5, xs:5}, height:{lg:'52%'}, bgcolor: "#f5f5f5",}}>
```

Comment:
change p to p:3
change bgcolor to backgroundColor

Reply:
maybe add in a comment tracking the height is set at 52% and what it is paired with